exec/java-exec/src/main/resources/drill-module.conf

```diff
@@ -120,7 +120,7 @@ drill.exec: {
     affinity.factor: 1.2
   },
   sys.store.provider: {
-    class: "org.apache.drill.exec.store.sys.zk.ZkPStoreProvider",
+    class: "org.apache.drill.exec.store.sys.store.provider.ZookeeperPersistentStoreProvider",
```

Comment:
Upgrade and downgrade process can be affected by this change (e.g. when overridden conf. file has "..._ZkPStoreProvider_"), right?